src/Package/Impl/Options/R/RToolsOptionsPage.cs

```diff
@@ -38,7 +38,7 @@ public class RToolsOptionsPage : DialogPage {
         [CustomLocDisplayName("Settings_CranMirror")]
         [LocDescription("Settings_CranMirror_Description")]
         [TypeConverter(typeof(CranMirrorTypeConverter))]
-        [DefaultValue("0-Cloud [https]")]
+        [DefaultValue(null)]
```

Comment:
If nothing is specified, default was that ugly CRAN selection dialog. That's why we started setting default mirror in the first place. So if nothing is specified we should still set something otherwise the dialog will be back.